common.yml

```diff
@@ -619,6 +620,8 @@ functions:
         source_dir: src/github.com/mongodb/mongo-tools
         include:
           - "./release.*"
```

Comment:
```suggestion
```
I think we want to remove this now that we're not generating a `release.{deb,rpm}` file